internal/venice-test-common/src/main/java/com/linkedin/venice/unit/kafka/consumer/poll/AbstractPollStrategy.java

```diff
@@ -41,11 +42,11 @@ public AbstractPollStrategy(boolean keepPollingWhenEmpty, int maxMessagePerPoll)
     this.maxMessagePerPoll = maxMessagePerPoll;
   }
 
-  protected abstract Pair<TopicPartition, Long> getNextPoll(Map<TopicPartition, Long> offsets);
+  protected abstract Pair<PubSubTopicPartition, Long> getNextPoll(Map<PubSubTopicPartition, Long> offsets);
```

Comment:
Would you mind creating a dedicated class to hold to the `PSTP` and the `long` offset? We are trying to move away from `Pair`.

Reply:
Sounds good. I will try to create a dedicated class for this.